src/core/Akka/Serialization/Serialization.cs

```diff
@@ -393,6 +393,41 @@ public void AddSerializationMap(Type type, Serializer serializer)
             _serializerMap[type] = serializer;
         }
 
+        /// <summary>
+        /// Remove the serializer mapping for <see cref="Type"/> <paramref name="type"/> and all other types
+        /// that were derived from it.
+        /// </summary>
+        /// <param name="type">The <see cref="Type"/> that will be removed from the serializer mapping</param>
+        /// <exception cref="InvalidOperationException">>Thrown when <paramref name="type"/> is of type <see cref="Object"/></exception>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void RemoveSerializationMap(Type type)
```

Comment:
Can we make this `internal` instead of `public` so this can't be abused by end-users?

Reply:
Good point